dist/livereload.js

```diff
@@ -385,19 +385,31 @@
     LiveReload.prototype.performReload = function(message) {
       var _ref, _ref1;
       this.log("LiveReload received reload request: " + (JSON.stringify(message, null, 2)));
-      return this.reloader.reload(message.path, {
```

Comment:
Please don't send `dist/livereload.js` changes in pull requests; they tend to create an ungodly amount of merge conflicts.